x/multi-staking/keeper/lock.go

```diff
@@ -25,7 +28,25 @@ func (k Keeper) EscrowCoinFrom(ctx context.Context, fromAcc sdk.AccAddress, coin
 }
 
 func (k Keeper) UnescrowCoinTo(ctx context.Context, toAcc sdk.AccAddress, coin sdk.Coin) error {
-	return k.bankKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleName, toAcc, sdk.NewCoins(coin))
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
+	err := k.bankKeeper.SendCoinsFromModuleToAccount(ctx, types.ModuleName, toAcc, sdk.NewCoins(coin))
```

Comment:
is it possible this will be commited and the unescrow fails?

Reply:
no since we send coin from module to account then send them out to `erc20` module directly with same amount.